### DISCUSSION TITLE
Is this working now

### DISCUSSION THREAD

**Discussion by kam5iz** [jonamat/wikiloc-earth-layer#5] [General]
Hello there
I used to use this last year but now trails are counted but not showed up either on map or in Places section ?
Thanks alot

**Comment by jonamat** [comment]
Hi! 👋
It's been a while since I last used the tool, I tried it now and I experienced the same issue you mentioned. It appears that the tool is able to fetch the trails from wikiloc correctly, but it is unable to display them on the map. It's possible that Google has made some changes to the KML parser in Earth Pro, causing this problem. I'll take a look and fix it as soon as possible!

**Comment by jonamat** [comment]
Fixed! Thanks for reporting this. Try to remove the place and reopen the kmz. Have fun!

**Comment by kam5iz** [comment]
Wow. Thanks a-lot
Amazing tool ! beyond facilitation
